index.html

```diff
@@ -131,34 +134,34 @@ <h1 class="title">
   <td><a href="https://github.com/webxdc/webxdc_docs">webxdc/webxdc_docs</a></td>
 </tr>
 <tr>
-  <td colspan="3"><a href="https://github.com/orgs/webxdc/repositories?type=all">existing apps</a></td>
-</tr>
-<tr>
-  <td>hello world</td>
-  <td></td>
-  <td><a href="https://github.com/webxdc/hello">webxdc/hello</a></td>
+  <td>examples</td>
+  <td><a href="https://github.com/webxdc/hello">hello world</a></td>
+  <td><a href="https://github.com/orgs/webxdc/repositories?type=all">all</a></td>
```

Comment:
```suggestion
  <td><a href="https://github.com/orgs/webxdc/repositories?type=all">more webxdc</a></td>
```

I try to use unique link text for accessibility reasons.